### ISSUE TITLE
Incorrect prompt after leaving an SSH session

### ISSUE THREAD

**Issue by jasonappah** [warpdotdev/Warp#1449]
### Discord username (optional)

jasonaa#7560

### Describe the bug

```text
After disconnecting from an SSH session, the user/host of the SSH server is still displayed in the prompt.
```

### To Reproduce

Steps to reproduce:
1. Log into another host with SSH
2. Exit the session
3. SSH user and server host remain in the prompt despite being disconnected from the server

### Screenshots

https://user-images.githubusercontent.com/30608521/171940915-d170edfb-31ba-4e14-bad5-26cf910b5912.mov

### Operating System

MacOS

### OS Version

12.4

### Shell Version

zsh 5.8.1 (arm-apple-darwin21.3.0)

### Warp Version

v0.2022.05.30.09.10.stable_01

**Comment by lixshnew** [comment]
I found same issue.

~
ssh host
name@host:~
exit
Connection to host closed.

name@host:~                          <---- this prompt is wrong, it will get back after you run a command like 'pwd'
pwd
/Users/name

~                                               <----- back to normal

Arm64
v0.2022.05.30.09.10.stable_02
Mac 12.4 (M1)

**Comment by renkun-ken** [comment]
Same problem here. Quite annoying and misleading.

<img width="303" alt="image" src="https://user-images.githubusercontent.com/4662568/173257924-2209be5f-d03e-45a2-b4a1-7dde59cc514b.png">

**Comment by lixshnew** [comment]
It was fixed on v0.2022.06.13.09.15.stable_00.  it works on my macos.
